content/300-guides/050-database/880-supabase.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Using Prisma with Supabase'
+metaTitle: 'Using Prisma with Supabase'
+metaDescription: 'Guide to using Prisma with Supabase'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+This guide discusses the concepts behind using Prisma and Supabase, explains the commonalities and differences between Supabase and other database providers, and leads you through the process for configuring your application to integrate with Supabase.
+
+</TopBlock>
+
+## What is Supabase?
+
+Supabase is an open source Firebase alternative providing all the backend features you need to build a product. Unlike Firebase, Supabase is backed by PostgreSQL which can be accessed directly.
+
+To learn more about Supabase, you can check out their architecture [here](https://supabase.com/docs/guides/getting-started/architecture) and features [here](https://supabase.com/docs/guides/getting-started/features)
+
+## Commonalities with other database providers
+
+Many aspects of using Prisma with Supabase are just like using Prisma with any other relational database. You can still:
+
+- model your database with the [Prisma Schema Language](/concepts/components/prisma-schema)
+- use Prisma's existing [`postgresql` database connector](/concepts/database-connectors/postgresql) in your schema, along with the [connection string Supabase provides you](https://supabase.com/docs/guides/database/connecting-to-postgres#finding-your-connection-string)
+- use [Introspection](/concepts/components/introspection) for existing projects if you already have a database schema in Supabase
+- use [`db push`](/concepts/components/prisma-migrate/db-push) to push changes in your schema to Supabase
+- use [Prisma Client](/concepts/components/prisma-client) in your application to talk to the database server at Supabase
+
+## Getting Started with Supabase
```

Comment:
Since we typically use sentence casing for section headings, this should be lowercased

```suggestion
## Getting started with Supabase
```